### ISSUE TITLE
E701 type hinting with attribute name starting with 'for'

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#1076]
In GitLab by @VincentKetelaars on Dec 19, 2017, 21:14

*Please describe how you installed Flake8*
```
$ pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*
```
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "28.8.0"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.2",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.3.1"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "1.6.0"
    }
  ],
  "version": "3.5.0"
}
```

*Please describe the problem or feature*
When running `flake8` on a simple `*.py` file that looks like this:

```
class Test:
    test: str
    format: str
```

The third line will give off a warning: `E701 multiple statements on one line (colon)`
The only cases that I have found that will result in this warning, is when the attribute starts with 'for'.

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*
I expect this line to be free of any warnings

**Comment by asottile** [comment]
In GitLab by @ben.samuel on Mar 5, 2018, 08:02

I was going to submit the same bug, but I believe this is addressed upstream in https://github.com/PyCQA/pycodestyle/pull/640

**Comment by asottile** [comment]
In GitLab by @ostrokach on Apr 20, 2018, 15:09

The issue may have been fixed with `pycodestyle=2.4.0`, but flake8 does not work with `pycodestyle=2.4.0`. I get an error:

```
flake8.exceptions.FailedToLoadPlugin: Flake8 failed to load plugin "pycodestyle.break_around_binary_operator" due to module 'pycodestyle' has no attribute 'break_around_binary_operator'.
```

**Comment by asottile** [comment]
In GitLab by @ostrokach on Apr 20, 2018, 15:15

Related issue: #415.

**Comment by asottile** [comment]
In GitLab by @jiim_e on Apr 21, 2018, 12:39

@ostrokach cf my comment on #406: https://gitlab.com/pycqa/flake8/issues/406#note_68880750 

Basically the fixes are ready but haven't been released yet.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 20, 2018, 10:18

http://flake8.pycqa.org/en/latest/internal/contributing.html#filing-a-bug

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 20, 2018, 10:18

closed
